### DISCUSSION TITLE
Is there a XML tag highlight setting based on where the cursor is located

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4019] [Q&A]
I was wondering if anyone knows if there is a XML tag highlight setting, based on where the cursor is located?
In other programs I have seen this handle a few different ways
-Some highlight the tag & close tag if the cursor is in the actual tags text only
-Some highlight the tag & close tag if the cursor is in the text data between the tags only
-Some highlight the tag & close tag when the cursor is in either of the items mentioned above.

I have also seen different approaches to how the highlight indicators look
-Some put a rectangular outline around the tags
-Some put a colored background highlight

Again just wondering if Geany contains anything similar to any of the things mentioned above.
Thank You

**Comment by elextr** [comment]
Check the Geany-plugins which has a number of plugins that add HTML/XML/etc functions.

**Comment by advice2020** [comment]
Okay, thank you
